service/frontend/adminHandler.go

```diff
@@ -597,24 +597,22 @@ func (adh *adminHandlerImpl) DescribeCluster(
 
 		var rings []*types.RingInfo
 		for _, role := range service.List {
-			resolver, err := monitor.GetResolver(role)
-			if err != nil {
-				return nil, adh.error(err, scope)
```

Comment:
Seems like we no longer return this error. Is this intended?

Reply:
thanks, fixed